CommandDotNet.Tests/FeatureTests/Arguments/AppsSettings_BoolMode.cs

```diff
@@ -33,8 +33,8 @@ public AppsSettings_BoolMode(ITestOutputHelper output) : base(output)
   operand
 
 Options:
-  -h | --help  Show help information
-  --option"
+  --option
+  -h | --help  Show help information"
```

Comment:
took the opportunity to move help and version to the bottom of the list.  when i'm looking for help for a command, i'm already seeing it and i'm more interested in the help specific to the command so moving this down reduces the noise